front/packages/ui/src/browse/header.tsx

```diff
@@ -60,79 +50,110 @@ const SortTrigger = forwardRef<View, PressableProps & { sortKey: string }>(funct
 	);
 });
 
+const MediaTypeTrigger = forwardRef<View, PressableProps & { mediaType: MediaType }>(
+	function MediaTypeTrigger({ mediaType, ...props }, ref) {
+		const { css } = useYoshiki();
+		const { t } = useTranslation();
+		const labelKey =
+			mediaType !== MediaTypeAll ? `browse.mediatypekey.${mediaType.key}` : "browse.mediatypelabel";
+		const icon = mediaType !== MediaTypeAll ? mediaType?.icon ?? FilterList : FilterList;
+		return (
+			<PressableFeedback
+				ref={ref}
+				{...css({ flexDirection: "row", alignItems: "center" }, props as any)}
+				{...tooltip(t("browse.mediatype-tt"))}
+			>
+				<Icon icon={icon} {...css({ paddingX: ts(0.5) })} />
+				<P>{t(labelKey as any)}</P>
+			</PressableFeedback>
+		);
+	},
+);
+
 export const BrowseSettings = ({
 	availableSorts,
 	sortKey,
 	sortOrd,
 	setSort,
+	availableMediaTypes,
+	mediaType,
+	setMediaType,
 	layout,
 	setLayout,
 }: {
 	availableSorts: string[];
 	sortKey: string;
 	sortOrd: SortOrd;
 	setSort: (sort: string, ord: SortOrd) => void;
+	availableMediaTypes: MediaType[];
+	mediaType: MediaType;
+	setMediaType: (mediaType: MediaType) => void;
 	layout: Layout;
 	setLayout: (layout: Layout) => void;
 }) => {
 	const { css, theme } = useYoshiki();
 	const { t } = useTranslation();
-	const filters: string[] = [];
-	// TODO: implement filters in the front.
 
 	return (
-		<View
-			{...css({
-				flexDirection: "row-reverse",
-				alignItems: "center",
-				marginX: ts(4),
-				marginY: ts(1),
-			})}
-		>
-			{filters.length !== 0 && (
+		<>
```

Comment:
This fragment is now usless and can be removed

Reply:
Good catch, thanks.